buildscripts/import-control.xml

```diff
@@ -38,6 +38,12 @@ General guidelines on imports:
     <allow pkg="io.opencensus.internal"/>
     <allow pkg="io.opencensus.tags"/>
   </subpackage>
+  <subpackage name="metrics">
+    <allow pkg="io.opencensus.common"/>
+    <allow pkg="io.opencensus.internal"/>
```

Comment:
metrics shouldn't use the internal package if it is in a separate artifact.

Reply:
Done.